src/lib/components/Timeline.svelte

```diff
@@ -2,10 +2,14 @@
 	import type { ActionData } from '$lib/types';
 	import Action from './Action.svelte';
 
-	let { actions = $bindable() }: { actions: ActionData[] } = $props()
+	let { actions = $bindable(), displaying}: { actions: ActionData[], displaying:boolean} = $props();
 </script>
 
-<div class="flex flex-col items-center h-[80vh] bg-btn_grey text-text_white p-1 rounded gap-2 w-80">
+<div
+	class="flex flex-col items-center h-[500px] bg-btn_grey text-text_white p-1 rounded gap-2 w-80 border-2 border-outline_gray
+	{displaying ? "hidden" : ""}"
```

Comment:
1. the position should be absolute because it should be floating over the contents
2. it should be transformed off the screen when not displayed (so we can animate it)

Reply:
something like
```
class="flex flex-col items-center bg-btn_grey text-text_white p-1 gap-2 fixed h-[90svh] inset-x-0 bottom-0
{displaying ? "translate-y-full" : ""}"